README.md

```diff
@@ -60,3 +60,16 @@ These are directions to build the app with EAS and then publish to testflight. T
 The app store metadata is saved within the `store.config.json` file. If this file is changed, you will need to push the updates to the app store using `eas metadata:push`
 
 An important thing to remember from the above example is the configVersion property. It helps with versioning changes that are not backward compatible.
+
+## Deep links
+
+### iOS
+
+Deep links are links that will open up in the mobile app instead of the browser. These links are defined in the apple-app-site-association (AASA) file, which is hosted on the main wavlake.com website.
+
+The following is the process for adding/removing/editing which paths will open in the app and which will not:
+
+1. Update the AASA file hosted at `wavlake.com/.well-known/apple-app-site-association`
+1. Update the [DeepLinkHandler](components/DeepLinkHandler.tsx) component (if needed). This is responsible for auto-redirecting the user to the proper page in the app.
+1. Generate a new build for the iOS app
+1. Install the new build (or push the update to the app store). This will trigger the app to re-fetch the updated AASA file.
```

Comment:
Fix numbering
```suggestion
2. Update the [DeepLinkHandler](components/DeepLinkHandler.tsx) component (if needed). This is responsible for auto-redirecting the user to the proper page in the app.
3. Generate a new build for the iOS app
4. Install the new build (or push the update to the app store). This will trigger the app to re-fetch the updated AASA file.
```